Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -108,6 +112,21 @@ class BrowserViewController: UIViewController {
         }
     }
 
+    override func willRotateToInterfaceOrientation(toInterfaceOrientation: UIInterfaceOrientation, duration: NSTimeInterval) {
```

Comment:
Instead of using willRotate/didRotate, we should use viewWillTransitionToSize since these methods are now deprecated. Doesn't look like we already override the method so feel free to add it. To replicate the didRotate part, you can use the completion handler of the coordinator passed into the method. I believe there's a method named

`coordinator.animateAlongsideTransition {}, completion {}`